packages/core/src/components/tooltip/tooltip.tsx

```diff
@@ -166,7 +166,7 @@ export class Tooltip implements IxOverlayComponent {
   }
 
   private async computeTooltipPosition(target: Element) {
-    const computeResponse = await computePosition(target, this.hostElement, {
+    return await computePosition(target, this.hostElement, {
```

Comment:
```suggestion
    return computePosition(target, this.hostElement, {
```

Await not needed here